sdk/src/message.rs

```diff
@@ -84,36 +76,49 @@ pub struct Message {
     /// signatures must match the first `num_required_signatures` of `account_keys`.
     pub num_required_signatures: u8,
 
+    /// The number of credit-only signed accounts. The last `num_credit_only_signed_accounts` of
+    /// the signed keys will be cached during runtime to allow parallel processing across
```

Comment:
The transaction documentation doesn't need to document the runtime implementation. It should be documented from the perspective of the client. What can the client observe? How about something like, "The last `num_credit_only_signed_accounts`  of the signed keys are credit-only accounts. Programs may process multiple transactions that add lamports to the same credit-only account within a single PoH entry, but are not permitted to debit lamports or modify account data. Transactions targeting the same debit account are evaluated sequentially. "